posts/inside-rust/2019-12-17-bisecting-rust-compiler.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title: "Bisecting Rust Compiler Regressions with cargo-bisect-rustc"
+author: Santiago Pastorino
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Let's say that you've just updated the Rust compiler version and have
+tried to compile your application and see a failure that wasn't there
+before. That's likely due to a regression in the compiler.  We've just
+released
```

Comment:
```suggestion
released a new version of
```

Maybe?  It might be a little confusing since the tool has been around for a few years.

Reply:
It was literally just released. The tool existed on the github repo but didn't exist on crates.io. Anyway it may worth explaining something about it?.